src/App.js

```diff
@@ -1,13 +1,36 @@
-
 import './App.css';
 import EmailCardList from "./EmailCardList/EmailCardList.js";
 import Header from './Header/Header.js'
 import SideBar from './SideBar/SideBar.js'
-import {useEffect, useState} from "react"
+import DisplayReadingPanel from "./ReadingPanel";
+import {useState, useEffect} from "react";
 
 const App = () => {
     const [allEmailSnippets, setAllEmailSnippets] = useState([])
+    const [emailSearchId, setEmailSearchId] = useState(98)
+    const [emailDataById, setEmailDataById] = useState([])
+    const [emailRepliesBoolean, setEmailRepliesBoolean] = useState(false)
     const [sidebarIsHidden, setSidebarIsHidden] = useState(true)
+    const [readingPanelDisplay, setReadingPanelDisplay] = useState(false)
+
+    const fetchEmailById = async () => {
+        let queryString = 'http://localhost:8080/emails/' + emailSearchId
+        if (emailRepliesBoolean) {
+            queryString = queryString + '?replies=1'
+        }
+        const EmailById = await fetch(queryString)
```

Comment:
1. this constant should not start with uppercase because it is not a component
2. `emailById` is an odd name, maybe just `emailData`?
3. then if you rename this one, you should also rename `jsonEmailById` below it